### PR TITLE
Update TPU docs for installation

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Ref: https://github.com/pytorch/xla/pull/2813

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/advanced/tpu.rst [hunk 1]
```diff
@@ -64,8 +64,7 @@ To get a TPU on colab, follow these steps:
 
    .. code-block::
 
-        !curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py
-        !python pytorch-xla-env-setup.py --version 1.7 --apt-packages libomp5 libopenblas-dev
+        !pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.8-cp37-cp37m-linux_x86_64.whl
```
Comment (Borda) [inline docs/source/advanced/tpu.rst:67]: well here the XLA version shall be aligned with PT version, right?

Comment (kaushikb11) [reply docs/source/advanced/tpu.rst:67]: The instructions are for Colab & they has updated the runtime to use torch>=1.8.0

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]

File: docs/source/advanced/tpu.rst (hunk 1 above)
Comment (lezwon) [reply docs/source/advanced/tpu.rst:67]: does this same command work for kaggle?

Comment (kaushikb11) [reply docs/source/advanced/tpu.rst:67]: @lezwon Just checked. It's 1.7 for Kaggle notebooks.
We could add `{torch.__version__}` to the URL as a placeholder.

**Pull request merged**